### PR TITLE
Issue 8967: Fix avatar in top bar

### PR DESCRIPTION
**Pull request by annando**
Fixes https://github.com/friendica/friendica/issues/8967

### CODE REVIEW

File: view/theme/frio/theme.php [hunk 1]
```diff
@@ -195,8 +195,7 @@ function frio_remote_nav($a, &$nav)
 	// this isn't optimal because the contact query will be done now twice
 	$fields = ['id', 'url', 'avatar', 'micro', 'name', 'nick', 'baseurl'];
 	if (local_user() && !empty($a->user['uid'])) {
-		$remoteUser = Contact::getById($a->user['uid'], $fields);
-		$remoteUser['name'] = $a->user['username'];
+		$remoteUser = DBA::selectFirst('contact', $fields, ['uid' => $a->user['uid'], 'self' => true]);
```
Comment (MrPetovan) [inline view/theme/frio/theme.php:198]: Oof.

Comment (MrPetovan) [inline view/theme/frio/theme.php:198]:
```suggestion
		$remoteUser = Contact::selectFirst($fields, ['uid' => $a->user['uid'], 'self' => true]);
```

**Comment by AlfredSK** [comment]
Is it ready to merge?

**Comment by annando** [comment]
I hope that @MrPetovan will merge it, since he approved it. I guess he approved it while the test still ran.

**Comment by MrPetovan** [comment]
I was waiting for the tests and went to bed.

**Comment by annando** [comment]
I thought so. I often have a look at the clock and am thinking how late it is at your place.

**Pull request merged**